base/complex.jl

```diff
@@ -178,6 +178,10 @@ bswap(z::Complex) = Complex(bswap(real(z)), bswap(imag(z)))
 
 isequal(z::Complex, w::Complex) = isequal(real(z),real(w)) & isequal(imag(z),imag(w))
 
+in(x::Complex, r::Range{T}) where {T<:Integer} =
+    isinteger(x) && !isempty(r) && real(x) >= minimum(r) && real(x) <= maximum(r) &&
+        (mod(real(x), step(r)) - mod(first(r), step(r)) == 0)
```

Comment:
The equivalent method below for `x::Real` converts `x` to type `T` prior to the `mod`. Should that be done here as well?